composer.json

```diff
@@ -26,14 +26,14 @@
         }
     ],
     "require": {
-        "php": "7.4.*",
+        "php": "^7.2 || ^8.0",
```

Comment:
We cannot drop the PHP Version lower then 7.4 because we are using typed properties

Reply:
Thanks! Must have changed since I opended this PR.